app/s2i/src/main/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fabric8/s2i-java:3.0-java8
+FROM fabric8/s2i-java:3.1-java11
```

Comment:
This implies that also standard S2I (springboot) runtime will change to java 11. We could either have a different base image for camel-k runtime, but I guess we don't want to maintain two different base images.

Reply:
yes the image has JDK11 then also for Spring Boot runtime, but the Maven POM that runs the integration explicitly sets the source and target Java version to 1.8.